### DISCUSSION TITLE
Consistent dataset download location

### DISCUSSION THREAD

**Discussion by Mattdl** [ContinualAI/avalanche#470] [Feature Request] [answered]
Hi! I've noticed that when running unit tests, examples or by default, the directories to download the raw datasets varies (examples/data, avalanche_root/data, ~/.data). As this is just the raw data, I think it would be helpful if all of them would use the same directory.

Also I'm not sure if this is possible already, but it would be helpful if users could just provide the root path for the ./data directory. Giving more flexibility on where to store the datasets. I think for larger datasets like MiniImagenet this is already possible, but for consistency, this would be helpful if all datasets could just be contained in the same location.

**Comment by vlomonaco** [comment, marked as answer]
Hi @Mattdl you are totally right, I thought we discussed this already with @lrzpellegrini and @ggraffieti. We will fix this asap.

**Reply by ggraffieti** [reply]
I think the problem is mainly with tests. 
By default all the datasets should be downloaded into `~/.avalache`, in order to download them only one time and use them in many different projects. 
However, in some tests, we left the download path `./data`. 
We should definitely change that, I can do it immediately.

**Reply by vlomonaco** [reply]
done in #477
